Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -745,64 +744,23 @@ class CombinatorialKalmanFilter {
         const CkfTypes::BranchVector<TrackIndexType>& newTrackStateList =
             *tsRes;
 
-        Result<std::pair<unsigned int, unsigned int>> procRes =
```

Comment:
This and many other changes belong to #3474, I suppose, If the inclusion of these changes is intended than this is fine.

Reply:
Right I put this in here to check physmon